### ISSUE TITLE
General Questions/Feedback

### ISSUE THREAD

**Issue by lmasseng** [GSA/ficam-arch#167]
I am not sure if these comments fall under "citizen-facing", but I wanted to submit them nonetheless.

Under Identity Management:
“Identity Management is how an agency uses attributes to establish and maintain enterprise identities for **employees and contractors.”** 
•	Will this framework cover Agency affiliates/volunteers/etc.?

Under Credential Management:
“You might use a PIV credential that includes a picture, the issuing agency logo, and cryptographic key pairs to assert your identity at a federal facility.” 
•	Will this framework cover PIV-I credentials? Similarly, what about CAC cards (DoD)?

Under Identity Management:
“Deactivate or **remove** identity records.
Keywords: Identity Lifecycle Management, Suspension, Archiving, **Deletion**”
•	Are the terms “removal” and “deletion” in reference to HR systems and databases? Identity records should not be deleted unless there is a duplicate record, and/or they have been marked for deletion and approved by the appropriate party (e.g. the PIV card management GSA USAccess program). Alternatively, the rest of the framework uses the language of access revocation and record deactivation or termination.

**Comment by lachellel** [comment]
@lmasseng Thank you for the comments!  These comments do fall under _enterprise_ versus consumer / public / citizen.   We'll include these in the review and updates.

**Comment by lmasseng** [comment]
Great! Thank you!

**Comment by sarahdobson** [comment]
Here are the changes we made! Thanks again for your feedback on this update of the FICAM Architecture.

**Introduction:**
Updated the introduction text to better explain the scope of the Architecture: "This version of the FICAM Architecture encompasses the enterprise ICAM policies, technologies and system approaches for government employees, contractors, and authorized partners. Citizen interactions with the federal government - or consumer ICAM - are not covered under this version of the FICAM Architecture."

**Identity Management:**
Improved the language on this page to clearly explain the scope of the Architecture: "Identity Management is how an agency collects, verifies, and manages attributes to establish and maintain enterprise identities for federal government employees, contractors, and authorized mission partners. This service does not apply to public or consumer identity management."

Clarified the Deactivation description to cover enterprise identities. Agencies own and manage the identities for the enterprise users, and sync data with the credentialing service providers. The USAccess program is not included in this service area.

**Credential Management:**
Updated text to include a reference to CAC: "You might use an agency-issued smart card, such as a PIV or CAC, that includes a picture and cryptographic key pairs to assert your identity at a federal facility."
